Cargo.toml

```diff
@@ -37,6 +37,7 @@ reqwest = "0.11.20"
 wasmedge-sdk = {  version = "=0.13.2", features = ["standalone"] }
 wasmedge-sys = "=0.17.5"
 wasmedge-types = "=0.4.4"
+
 # https://github.com/denoland/deno_ast/blob/main/Cargo.toml
 swc_atoms = "=0.6.5"
```

Comment:
do we need to keep them?

Reply:
The typescript codegen still depend on swc_*, but it should be removable once the typescript mdk finalization lands.